scripts/ipfs-pi-stream/install

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -e
+
+BASE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# Enable camera on the Raspberry Pi
+sudo $BASE_DIR/enable-camera.sh
+
+# Install ffmpeg and supporting tools
+sudo apt-get -y install ffmpeg lsof inotify-tools nginx
+
+# Add user to be able to modify nginx directories
+sudo usermod -a -G $USER www-data
+sudo chmod g+rw /var/www/html
+
+# TODO: why is this needed?
+sudo chmod a+rw /var/www/html
+
+sudo cp -f $BASE_DIR/process-stream.sh /usr/bin/process-stream.sh
+sudo cp -f $BASE_DIR/process-stream.service /etc/systemd/system/process-stream.service
+sudo systemctl daemon-reload
+sudo systemctl enable process-stream
```

Comment:
I'm not sure if enabling this is a good idea, since it means a live stream is being constantly recorded since boot, which isn't really made obvious in the install script or the readme. The user can always disable it of course, but I would at least mention it somewhere.

Reply:
You also get a nice bright light in front of the camera telling you its recording.  I wanted a 0 fuss setup of this.

However adding it to the docs does make sense i think

Reply:
I didn't know about the light, and I understand that this isn't the final, secure, deployment product. Adding it to the docs sounds good to me!